Makefile

```diff
@@ -106,4 +106,4 @@ $(OBJDIR):
 	mkdir -p $(OBJDIR)
 
 clean:
-	rm -f $(OBJDIR)/*.o $(OBJDIR)/ccx_2.13.a $(OBJDIR)/ccx_preCICE
+rm -f $(OBJDIR)/*.o $(OBJDIR)/ccx_2.13.a $(OBJDIR)/ccx_preCICE
```

Comment:
Watch out: Indentation in `Makefiles` is crucial. Deleting the tab here creates a `Makefile` that is not functioning. Please redo this such that there are no changes at the `Makefile` at all.